lib/models/movie.dart

```diff
@@ -0,0 +1,34 @@
+class Movie {
+  final String title;
+  final String originalTitle;
+  final String overview;
+  final DateTime? releaseDate;
+  final String poster;
+  final String backDrop;
+  final double rate;
+  final List<String> genres;
```

Comment:
genres is defined as a List<int>

Reply:
Right! I made a GenreRepository to replace this, now it's a List<int> called genres_ids !